ares/ng/card/slot.cpp

```diff
@@ -9,6 +9,7 @@ auto CardSlot::load(Node::Object parent) -> void {
   port->setType("Memory Card");
   port->setHotSwappable(true);
   port->setAllocate([&](auto name) { return allocate(name); });
+  port->setDisconnect([&] { device.reset(); });
```

Comment:
This is not a controller, though it is a designated "HotSwappable" device, albeit one without any associated UI. I included this for completeness, but I could just as easily leave it out, because it currently has no effect.